react/features/participants-pane/components/native/MeetingParticipantList.js

```diff
@@ -151,7 +148,7 @@ class MeetingParticipantList extends PureComponent<Props> {
                 }
                 <FlatList
                     bounces = { false }
-                    data = { [ _localParticipantId, ..._sortedRemoteParticipants ] }
```

Comment:
Why is the local participant out of the data set?

Reply:
Because it's actually part of the sortedRemoteParticipants array. I'll rename it